src/Game.css

```diff
@@ -0,0 +1,67 @@
+.Game {
+}
+
+.Game-description {
```

Comment:
Remove unused code throughout your whole app. And make sure to always name CSS class names in kebab-case

Reply:
(lowercase)